CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+## Unreleased
+
+### Improvements
+
+- Update `io.github.gradle-nexus.publish-plugin` to 2.0.0 from 1.1.0.
+
```

Comment:
If you add this to `CHANGELOG_PENDING` it gets taken care of for you as part of the release process.

Reply:
My kingdom for a consistent changelog process :)